File: FrontEnd/src/app/features/schedules/classes/form/schedule-read-dto.ts
import { DestinationDropdownDTO } from './../../../destinations/classes/dtos/destination-dropdown-dto'
import { PortDropdownDTO } from './../../../ports/classes/dtos/port-dropdown-dto'

export class ScheduleReadDTO {

    id: number
    destination: DestinationDropdownDTO
    port: PortDropdownDTO
    date: string
    maxPassengers: number
    isActive: boolean

}

File: API/Features/Schedules/Dtos/ScheduleReadDto.cs
using API.Infrastructure.Classes;

namespace API.Features.Schedules {

    public class ScheduleReadDto {

        public int Id { get; set; }
        public string Date { get; set; }
        public SimpleResource Destination { get; set; }
        public SimpleResource Port { get; set; }
        public int MaxPassengers { get; set; }
        public string DepartureTime { get; set; }
        public bool IsActive { get; set; }

    }

}